### DISCUSSION TITLE
Enable/Disable Problem with Docker Intallation

### DISCUSSION THREAD

**Discussion by BurakcanKolay** [rahulhaque/soketi-app-manager-filament#5] [Q&A] [answered]
If the app is initially opened as active, it remains active, and if it's initially closed, it stays closed. Changing it later doesn't make a difference. Where should I look for the issue?
![image](https://github.com/rahulhaque/soketi-app-manager-filament/assets/26367826/6068e537-8cc8-44c0-94d6-238824b565cf)
![image](https://github.com/rahulhaque/soketi-app-manager-filament/assets/26367826/661668f5-025c-4309-9bfc-8e15ba39c9eb)

> **I have docker installation**

**Comment by BurakcanKolay** [comment, marked as answer]
I found the answer to the problem. The cache feature with a TTL of 3600 was active on the WebSocket container.
![image](https://github.com/rahulhaque/soketi-app-manager-filament/assets/26367826/496ebd5d-815c-4cca-b6e5-8230ae26a28a)

**Reply by rahulhaque** [reply]
@BurakcanKolay This is an issue I have faced myself. However didn't manage time to look up by myself to invalidate the app cache manually. Opened a discussion [here](https://github.com/soketi/soketi/discussions/1163). It would be nice to have official support for this rather than me hacking through. You can join the mentioned discussion so it gets noticed. Marking this as accepted answer for now till new solution arrives.
